platform/os_linux.go

```diff
@@ -105,3 +105,8 @@ func KillProcessByName(processName string) error {
 	_, err := ExecuteCommand(cmd)
 	return err
 }
+
+// Perform platform specific initialization
+func Init(createExtSwitchNetworkType string) error {
```

Comment:
Do you think it is appropriately named based on the input it takes?

Reply:
Changed in the latest commit.